### ISSUE TITLE
Execution failed for task ':dchs_motion_sensors:compileDebugKotlin'.

### ISSUE THREAD

**Issue by gputhige** [dariocavada/panorama_viewer#16]
I am getting this error all of a sudden which was not there earlier. 

Execution failed for task ':dchs_motion_sensors:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction

**Comment by talhaarshad25** [comment]
i am getting the same issues

**Comment by dariocavada** [comment]
Thanks for reporting this issue! To better understand and replicate the problem, could you please provide:

**Flutter Environment** – The output of `flutter doctor -v.`
**Logs & Error Messages** – Any relevant logs or error stack traces (`flutter run --verbose`).
**Platform & Version** – Which platform(s) are affected (Android/iOS/Web) and their version.

This will help us investigate and resolve the issue more efficiently. 
Thanks in advance!

**Comment by Mina-Nashaat10** [comment]
any updates ??

**Comment by dariocavada** [comment]
I'm waiting for more info to replicate the problem as:


> Thanks for reporting this issue! To better understand and replicate the problem, could you please provide:
> 
> **Flutter Environment** – The output of `flutter doctor -v.` **Logs & Error Messages** – Any relevant logs or error stack traces (`flutter run --verbose`). **Platform & Version** – Which platform(s) are affected (Android/iOS/Web) and their version.
> 
> This will help us investigate and resolve the issue more efficiently. Thanks in advance!

**Comment by ftaska** [comment]
Hi, i getting the same issue, here's my report:

**Flutter Environment**
[√] Flutter (Channel stable, 3.29.1, on Microsoft Windows [Version 10.0.22631.4890], locale en-ID) [423ms]
    • Flutter version 3.29.1 on channel stable at C:\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 09de023485 (10 days ago), 2025-02-28 13:44:05 -0800
    • Engine revision 871f65ac1b
    • Dart version 3.7.0
    • DevTools version 2.42.2

[√] Windows Version (11 Pro 64-bit, 23H2, 2009) [1,280ms]

[√] Android toolchain - develop for Android devices (Android SDK version 35.0.1) [1,916ms]
    • Android SDK at C:\Users\FXXX UXXXX\AppData\Local\Android\sdk
    • Platform android-35, build-tools 35.0.1
    • Java binary at: C:\Program Files\Android\Android Studio\jbr\bin\java
      This is the JDK bundled with the latest Android Studio installation on this machine.
      To manually set the JDK path, use: `flutter config --jdk-dir="path/to/jdk"`.
    • Java version OpenJDK Runtime Environment (build 21.0.5+-12932927-b750.29)
    • All Android licenses accepted.

[√] Chrome - develop for the web [138ms]
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop Windows apps (Visual Studio Build Tools 2019 16.11.43) [137ms]
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools
    • Visual Studio Build Tools 2019 version 16.11.35706.149
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2024.3) [13ms]
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 21.0.5+-12932927-b750.29)

[√] VS Code (version 1.98.1) [11ms]
    • VS Code at C:\Users\FXXXX UXXXX\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.106.0

[√] Connected device (4 available) [214ms]
    • U70C (mobile)     • 62235945604251 • android-arm64  • Android 8.1.0 (API 27)
    • Windows (desktop) • windows        • windows-x64    • Microsoft Windows [Version 10.0.22631.4890]
    • Chrome (web)      • chrome         • web-javascript • Google Chrome 133.0.6943.142
    • Edge (web)        • edge           • web-javascript • Microsoft Edge 133.0.3065.59

[√] Network resources [1,382ms]
    • All expected network resources are available.

• No issues found!

can i send my log & error message on email?
it affected Android 8.1.0 version
Thanks:)

**Comment by dariocavada** [comment]
Have you tried the latest version? I made some adjustments to the compilation process, so it should work now.

**Comment by ftaska** [comment]
No, I haven't sir. But thanks for information, I'll try to change and reply
a message maybe this weekend.

Pada Rab, 13 Agt 2025 21.42, Dario Cavada ***@***.***>
menulis:
